### PR TITLE
What's New 8.5

### PR DESCRIPTION
**Pull request by ollyhowell**
Closes https://github.com/elastic/observability-docs/issues/2218

### CODE REVIEW

File: docs/en/observability/whats-new.asciidoc [hunk 1]
```diff
@@ -3,15 +3,109 @@
...
 Here are the highlights of what's new and improved in {minor-version}.
 
-coming[8.0.0]
-
-// Other versions:
-// {observability-guide-all}/8.1/whats-new.html[8.1] |
-// {observability-guide-all}/8.0/whats-new.html[8.0] |
-// {observability-guide-all}/7.17/whats-new.html[7.17] |
+Other versions:
+{observability-guide-all}/8.4/whats-new.html[8.3] |
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:7]:
```suggestion
{observability-guide-all}/8.4/whats-new.html[8.4] |
```

File: docs/en/observability/whats-new.asciidoc [hunk 2]
```diff
@@ -3,15 +3,109 @@
...
+[role="screenshot"]
+image::images/hints-config-example.png[Example annotations configuration for redis]
+
+For more info, https://www.elastic.co/guide/en/fleet/master/elastic-agent-kubernetes-autodiscovery.html#_hints_annotations_based_autodiscover[read the docs].
+
+[discrete]
+== New Istio integration added
+
+With a new Istio integration available in beta for version 8.5, you can collect
+logs from the Istio service mesh with Elastic Agent.
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:52]: I normally don't mark this because it's such a pervasive problem, but the phrase "with Elastic Agent" here modifies "mesh" grammatically. I would say "by using" instead.

File: docs/en/observability/whats-new.asciidoc [hunk 3]
```diff
@@ -3,15 +3,109 @@
...
+
+The Istio service mesh extends Kubernetes to establish a programmable
+application-aware network using the powerful Envoy service proxy. Working with
+both Kubernetes and traditional workloads, Istio brings universal traffic
+management, telemetry, and security, to complex deployments. {integration-docs}/istio[Read the docs] for
+more information, and look out for support for metrics coming in future releases.
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:58]: Can you rephrase this? It parses awkwardly.

File: docs/en/observability/whats-new.asciidoc [hunk 4]
```diff
@@ -3,15 +3,109 @@
...
+
+[role="screenshot"]
+image::images/istio-integration.png[Add the Istio integration to collect logs from the Istio service mesh]
+
+[discrete]
+== View contextual Kubernetes info from APM Service Overview
+
+You can now view Kubernetes-related metadata in the context of a service. From
+the **Service Overview** page, click on the Kubernetes icon to the right of the
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:67]:
```suggestion
the **Service Overview** page, click the Kubernetes icon next to the
```

File: docs/en/observability/whats-new.asciidoc [hunk 5]
```diff
@@ -3,15 +3,109 @@
...
+application-aware network using the powerful Envoy service proxy. Working with
+both Kubernetes and traditional workloads, Istio brings universal traffic
+management, telemetry, and security, to complex deployments. {integration-docs}/istio[Read the docs] for
+more information, and look out for support for metrics coming in future releases.
+
+[role="screenshot"]
+image::images/istio-integration.png[Add the Istio integration to collect logs from the Istio service mesh]
+
+[discrete]
+== View contextual Kubernetes info from APM Service Overview
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:64]:
```suggestion
== View contextual Kubernetes info in the APM Service Overview
```
"from" makers it sound like the info is coming from the overview, not being viewed through it. I'm not sure you even need "in the APM Service Overview" here

File: docs/en/observability/whats-new.asciidoc [hunk 6]
```diff
@@ -3,15 +3,109 @@
...
+
+You can now view Kubernetes-related metadata in the context of a service. From
+the **Service Overview** page, click on the Kubernetes icon to the right of the
+service name to open a popup that contains Kubernetes details such as container
+images, deployments, namespaces, and replicasets.
+
+[role="screenshot"]
+image::images/apm-overview-kubernetes-context.png[View Kubernetes info from APM Service Overview]
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:72]:
```suggestion
image::images/apm-overview-kubernetes-context.png[View Kubernetes info in APM Service Overview]
```

File: docs/en/observability/whats-new.asciidoc [hunk 7]
```diff
@@ -3,15 +3,109 @@
...
+
+You can also view additional information from the **Instance detail** view at the bottom of the page.
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:74]:
```suggestion
You can also view additional information in the **Instance detail** view at the bottom of the page.
```

File: docs/en/observability/whats-new.asciidoc [hunk 8]
```diff
@@ -3,15 +3,109 @@
...
+
+[role="screenshot"]
+image::images/apm-services-instance-kubernetes.png[View Kubernetes info from APM Service Overview]
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:77]:
```suggestion
image::images/apm-services-instance-kubernetes.png[View Kubernetes info in APM Service Overview]
```

File: docs/en/observability/whats-new.asciidoc [hunk 9]
```diff
@@ -3,15 +3,109 @@
...
+image::images/apm-storage-explorer.png[View storage footprint with APM Storage Explorer]
+
+[role="screenshot"]
+image::images/apm-storage-explorer-details.png[View storage details for services]
+
+[discrete]
+== Auto-instrument APM .NET Agent profiler
+
+You can now dynamically instrument existing .NET Framework, .NET Core, and .NET
+applications during their startup and gain complete visibility in a matter of minutes.
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:98]:
```suggestion
You can now dynamically instrument existing .NET Framework, .NET Core, and .NET
applications during startup and gain complete visibility in a matter of minutes.
```

File: docs/en/observability/whats-new.asciidoc [hunk 10]
```diff
@@ -3,15 +3,109 @@
...
+SREs and DevOps engineers are now able to instrument
+existing .NET applications in their environments, whether running in containers,
+IIS, or as Windows or Linux services, without needing to recompile any of the
+binaries.
+
+In addition, the latest version of .NET agent extends distributed tracing support
+by providing auto-instrumentation for new libraries including Kafka, RabbitMQ,
+MySql, Oracle and PostgreSQL.
+
+For more info, https://www.elastic.co/guide/en/apm/agent/dotnet/current/setup-auto-instrumentation.html[read the docs].
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:108]: You need to use an attribute to resolve this path.

**Review by dedemorton — COMMENTED** [review]
Some minor edits. Otherwise looks good!

I wonder though if there's time to fix this diagram (the alignment is off): docs/en/observability/images/hints-based-autodiscovery.png

File: docs/en/observability/whats-new.asciidoc (hunk 1 above)
Comment (ollyhowell) [reply docs/en/observability/whats-new.asciidoc:7]: Ha, you're too good - I spotted that and fixed it locally but didn't think anyone would notice before I updated!

(I probably should have taken obs-docs off review until PMs had finished reviewing...)

File: docs/en/observability/whats-new.asciidoc [hunk 11]
```diff
@@ -3,15 +3,109 @@
...
+{observability-guide-all}/8.2/whats-new.html[8.2] |
+{observability-guide-all}/8.1/whats-new.html[8.1] |
+{observability-guide-all}/8.0/whats-new.html[8.0] |
+{observability-guide-all}/7.17/whats-new.html[7.17] |
 
 // tag::whats-new[]
 
-// What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)
+[discrete]
+== Perform annotations-based monitoring of Kubernetes workloads with standalone Elastic Agent
```
[the remark this answers is not in the document]
Comment (ollyhowell) [reply docs/en/observability/whats-new.asciidoc:17]: Thanks, hadn't got to that stage yet but fixing now!

File: docs/en/observability/whats-new.asciidoc [hunk 12]
```diff
@@ -3,15 +3,109 @@
...
+
+This feature enables Elastic operators to provide Kubernetes cluster users with
+a self-service monitoring configuration experience that is GitOps friendly.
+You can enable this feature in the Elastic Agent YAML file by uncommenting
+the init container as shown:
+
+[role="screenshot"]
+image::images/hints-autodiscovery-yaml.png[Uncomment to enable hints-based monitoring]
+
+You can download Elastic Agent templates for https://github.com/elastic/elastic-agent/tree/main/deploy/kubernetes/elastic-agent-standalone/templates.d[~60 supported packages]
```
[the remark this answers is not in the document]
Comment (ollyhowell) [reply docs/en/observability/whats-new.asciidoc:32]: Good question. I'd guess main is probably more useful in this context (and users can switch if needed) but what do you think @mlunadia ? Do these templates change much per version?

File: docs/en/observability/whats-new.asciidoc (hunk 9 above)
Comment (ollyhowell) [reply docs/en/observability/whats-new.asciidoc:98]: I did originally edit that out but then wasn't sure if there was a difference between startup of applications and startup of _something else_ / elastic so was waiting for some PM confirmation. Will take it out again unless I hear something different.

File: docs/en/observability/whats-new.asciidoc (hunk 12 above)
[the remark this answers is not in the document]
Comment (ollyhowell) [reply docs/en/observability/whats-new.asciidoc]: Will link to 8.5 in this case...
